### PR TITLE
Add the option to include shell scripts in data.json

### PR DESCRIPTION
**Pull request by tthbalazs**
Hey!

Tested with data that looks something like this:
```json
    {
      "id": "shell_scripts",
      "name": "Shell Scripts",
      "order": 1
    }

    {
      "id": "ls",
      "type": "shell_scripts",
      "name": "List directories",
      "description": "A simple command",
      "category": "shell_scripts",
      "code": "ls -la"
    },
    {
      "id": "defaults",
      "type": "shell_scripts",
      "name": "Configure defaults",
      "description": "A setting to allow full screen iOS Simulators",
      "category": "shell_scripts",
      "code": "defaults write com.apple.iphonesimulator AllowFullscreenMode -bool YES"
    },
```

With longer commands the cards can look ugly, also I don't know if there is a clear icon for this feature available or not :)

Anyway I thought I'd push the changes early so you can give feedback on the direction. Cheers!

### CODE REVIEW

File: src/pages/app.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ class App extends Component {
   }
 
   fetchData() {
-    const url = 'https://raw.githubusercontent.com/max-itup/content/master/mac/data.json'
+    const url = 'http://localhost:3000/data.json'
```
Comment (tthbalazs) [inline src/pages/app.js]: Oops, probably shouldn't have pushed this.

Comment (omaralbeik) [reply src/pages/app.js]: Yep, please remove this :)

File: src/utils/index.js [hunk 2]
```diff
@@ -47,6 +50,7 @@ export function objectFromArray(arr, key = 'id') {
     script = script.replace('{{PACKAGES}}', statics.brew(brew_items));
     script = script.replace('{{GEMS}}', statics.gems(gem_items));
     script = script.replace('{{OS_SETTINGS}}', statics.os_settings(os_settings_items));
+    script = script.replace('{{SHELL_SCRIPTS}}', statics.shell_scripts(shell_script_items));
     return script;
   }
```
Comment (omaralbeik) [inline src/utils/index.js:56]: Looking good 😎

**Review by omaralbeik — COMMENTED** [review]
All looking good, please remove the localhost url, and I'll merge it in no time 👍

**Comment by omaralbeik** [comment]
Thanks for contributing!, for the full-screen setting, we already have a [`os_settings`](https://github.com/max-itup/content/blob/128de2f00e99c97be97bad56781598bd877f1199/mac/data.json#L129) category to set OS settings, here is an [example](https://github.com/max-itup/content/blob/128de2f00e99c97be97bad56781598bd877f1199/mac/data.json#L1202).

**Review by omaralbeik — APPROVED** [review]
Thanks @tthbalazs!

**Pull request merged**